app/code/community/Convenient/CategoryCode/sql/convenient_categorycode_setup/mysql4-upgrade-0.1.0-0.1.1.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/* @var $installer Mage_Catalog_Model_Resource_Setup */
+$installer = $this;
+$installer->startSetup();
```

Comment:
@asif-ali Thanks for this.
Classic case of "do as I say, not as I do"

http://magento.stackexchange.com/questions/68387/usage-of-superfluous-and-potentially-dangerous-standard-code-in-installer-scri

can you remove the installer `startSetup` and `endSetup` please?